src/domain/app/appWideNotification/AppWideNotification.tsx

```diff
@@ -35,13 +35,15 @@ export function AppWideNotification() {
   const [isOpen, setOpen] = useState(() =>
     getInitialValue(sessionStorage.getItem(IS_OPEN_KEY), isNotificationEnabled)
   );
+
   const {
     t,
     i18n: { language },
   } = useTranslation();
 
-  const notificationContent = notificationContentTranslations[language];
-  const notificationTitle = notificationContentTitleTranslations[language];
+  const lang = language.slice(0, 2);
```

Comment:
trivial thing and just an opinion, but I think it would be better to use split instead of slice: `"fi-FI".split("-")[0]`

```suggestion
  const lang = language.split("-")[0];
```

It seems like the i18n-instance is configured with default and fallback languages, so I guess the language can never be null, so no null check needed - just thinking out loud :)